source/daplink/drag-n-drop/vfs_user.c

```diff
@@ -334,12 +353,22 @@ static uint32_t read_file_mbed_htm(uint32_t sector_offset, uint8_t *data, uint32
 // File callback to be used with vfs_add_file to return file contents
 static uint32_t read_file_details_txt(uint32_t sector_offset, uint8_t *data, uint32_t num_sectors)
 {
+    uint32_t size = 0;
 
-    if (sector_offset != 0) {
+    if (sector_offset > 1) {
```

Comment:
This should probably depend on the value of `BOARD_EXTRA_BUFFER`

Reply:
Updated calculation.